dvc/utils/__init__.py

```diff
@@ -237,18 +235,6 @@ def current_timestamp():
     return int(nanotime.timestamp(time.time()))
 
 
-def from_yaml_string(s):
-    return YAML().load(io.StringIO(s))
-
-
-def to_yaml_string(data):
-    stream = io.StringIO()
-    yaml = YAML()
-    yaml.default_flow_style = False
-    yaml.dump(data, stream)
-    return stream.getvalue()
-
-
```

Comment:
These weren't being used, seems leftover from `tags`. Moved these to `_yaml` and slightly changed the UI.